runtime.go

```diff
@@ -1098,14 +1098,19 @@ func (r *Runtime) EnableDebugMode() *Debugger {
 // method. This representation is not linked to a runtime in any way and can be run in multiple runtimes (possibly
 // at the same time).
 func Compile(name, src string, strict bool) (*Program, error) {
-	return compile(name, src, strict, false, true)
+	return compile(name, src, strict, false, true, false)
```

Comment:
This is what I did before, but at that time I didn't know about the `dynLookup` and `dynamic`. I also changed public APIs, which was a wrong move. :roll_eyes: